src/effects/Generator.cpp

```diff
@@ -71,7 +71,7 @@ bool Generator::Process(EffectInstance &, EffectSettings &settings)
                   ViewInfo::Get(*pProject).selectedRegion;
                track.ClearAndPaste(
                   selectedRegion.t0(), selectedRegion.t1(),
-                  *list, true, false, &warper);
+                  *list, true, true, &warper);
```

Comment:
Can you explain why this change is correct?

Reply:
`merge` will become `mergeIfPossible` in `PasteOne`. This preserves the old behavior if possible, i.e., if `selectedRegion` spans clips of stretch ratio 1.

Reply:
I don't accept that explanation.

Compare behavior before and after this PR:
- Generate noise.
- Split the clip.
- Select around the split.
- Generate silence.

There is a change of behavior, as shown below.  But I thought the intention, with effects and generators (unlike with copy/paste), and no stretches, was to change no behavior.

Result, before this PR:

<img width="1344" alt="image" src="https://github.com/audacity/audacity/assets/11670369/1ef9e0fa-f6b5-4387-bc37-6cffb04e5667">

After:

<img width="1344" alt="image" src="https://github.com/audacity/audacity/assets/11670369/29946a48-7bcf-469e-9e6a-892df57967ac">